presto-main/src/main/java/com/facebook/presto/execution/SqlQueryManager.java

```diff
@@ -312,4 +324,21 @@ private void enforceCpuLimits()
             }
         }
     }
+
+    /**
+     * Enforce query scan physical bytes limits
+     */
+    private void enforceScanLimits()
+    {
+        for (QueryExecution query : queryTracker.getAllQueries()) {
+            BasicQueryStats basicQueryStats = query.getBasicQueryInfo().getQueryStats();
+            QueryStats queryStats = query.getQueryInfo().getQueryStats();
```

Comment:
these 2 are not being used - remove these

Reply:
sure, it's removed